saleor/graphql/app/types.py

```diff
@@ -524,6 +535,7 @@ class App(ModelObjectType[models.App]):
         required=True,
     )
     brand = graphene.Field(AppBrand, description="App's brand data.")
+    breaker_state = graphene.String(description="App sync webhooks operation state.")
```

Comment:
This looks more like an enum in the schema. I also think the description is rather vague, and we should explicitly say that this is the state of the app's circuit breaker. If we had an enum in the schema, we could also describe what each state value exactly means.

I know that there was a suggestion not to use the "circuit breaker" term in the schema, but in this case, how should we call it to make sure that it's understandable to the user? @patrys Do you have any suggestion?